index.html

```diff
@@ -265,6 +265,45 @@ <h3>FindCollabs</h3>
           </p>
         </section>
       </div>
+      <section>
+        <h2>Donated Services</h2>
+        <p>
+          Looking for computing power, data storage or other tools and services
+          to power your project? Here's some services being offered for free to
+          anyone combatting COVID-19!
+        </p>
+        <h3>Fauna</h3>
+        <p>
+          <a href="https://fauna.com/"> FaunaDB</a> is a serverless cloud
```

Comment:
On other websites, I've noticed they explain if the offer is available globally, or just in certain regions. I assume that this doesn't apply here given the nature of the service but can you specify that there are no restrictions so we don't get questions?

Reply:
I added some more info from Fauna, should be _very_ clear whats being offered now

Reply:
I've checked this project out locally, and I do agree that Markdown may not serve us and participants well.  As a participant, I just want the tl;dr, and right now this doesn't scale with just 1 resource:

<img width="1967" alt="image" src="https://user-images.githubusercontent.com/6334517/78208795-8712e180-7462-11ea-862f-636fa7ee0e84.png">

It's a bit of a learning curve, and at the risk of making this PR take way too long to get shipped, could we consider making a Resources  index template in HTML and authoring each individual resource as a Jekyll [Collection item](https://jekyllrb.com/docs/step-by-step/09-collections/)?  I believe this will allow for:

1. The visual freedom to get something akin to [Hyperspace](https://html5up.net/uploads/demos/hyperspace/index.html)'s image+summary listview: 
  <img width="1967" alt="image" src="https://user-images.githubusercontent.com/6334517/78208953-f2f54a00-7462-11ea-88d1-bc1f09d787b2.png">
2. Allow public contributors to add another Jekyll resource via PR, without needing to worry about HTML.